chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -322,6 +322,21 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
             }
         });
 
+        BroadcastService.getMessageByEventName(BroadcastEvent.NetworkReconnected).subscribe(async () => {
+            if (isThisSessionPopout(window?.location?.href)) {
+                return;
+            }
+            const conversationDetails = await getConversationDetailsCall(chatSDK);
+
+            if (conversationDetails?.state === LiveWorkItemState.WrapUp || conversationDetails?.state === LiveWorkItemState.Closed) {
+                dispatch({ type: LiveChatWidgetActionType.SET_CHAT_DISCONNECT_EVENT_RECEIVED, payload: true });
+                TelemetryHelper.logActionEvent(LogLevel.INFO, {
+                    Event: TelemetryEvent.ChatDisconnectThreadEventReceived,
+                    Description: "Chat disconnected due to timeout, left or removed."
```

Comment:
Should the `Description` be more descriptive since it's related to `network reconnection`?